tests/nightly/test_large_array.py

```diff
@@ -17,7 +17,8 @@
 
 import numpy as np
 import mxnet as mx
-from mxnet.test_utils import rand_ndarray, assert_almost_equal, rand_coord_2d
+import math
```

Comment:
Move this import before numpy: http://google.github.io/styleguide/pyguide.html#313-imports-formatting